### PR TITLE
Issue 21 Add the option to choose between standalone example or standalone with git

### LINKED ISSUE #21: Add the option to choose between standalone example or standalone with git
## Feature description
Giving the option to choose which example to run: standalone or standalone with git integration. Default will be to run standalone when running `make up `
## Checklist (after created issue)
- [x] Added label(s)
- [ ] Added to project
- [x] Added to milestone

### PR DESCRIPTION
**Pull request by karimacherradi**
## Closes/fixes/resolves issue(s)?
Closes #21 
## What was added/changed/fixed?
- Updated Makefile with option to for nomad to switch to use standalone or standalone with git #21
- updated example/standalone/main.tf with the same switch option #21
- Updated variables.tf and main.tf with new mode-variable #21
- Updated dev/ansible/01_run_terraform.yml #21

## Checklist (after created PR)
- [x] Added reviewers
- [x] Added assignee(s)
- [x] Added label(s)
- [ ] Added to project
- [x] Added to milestone
- Need to update documentation? (Y/n)

### CODE REVIEW

**Comment by AditiSharma77** [comment]
could you please give a proper name of this PR , instead of Issue 21 fix standalone .

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
 ### Changed
+- Update vagrant-box, `version = ">= 0.9, < 0.10"` #17
+- Updated Makefile with option to for nomad to switch to use standalone or standalone with git #21
+- updated example/standalone/main.tf with the same switch option #21
+- Updated variables.tf and main.tf with new mode-variable #21
+- Updated dev/ansible/01_run_terraform.yml #21
 
 ## [0.1.0]
-
+git status
```
Comment (hannemariavister) [inline CHANGELOG.md]: I think this could be removed?

**Review by hannemariavister — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (karimacherradi) [reply CHANGELOG.md]: Fix now

**Review by hannemariavister — APPROVED** [review]
Good work 💯

**Pull request merged**